### PR TITLE
first run: skip account creation if they already have accounts

### PR DESCRIPTION
**Pull request by derhuerst**
This PR addresses #3583 by adding a simple "skip" button to the first run modal, if the user has one or more accounts.

### CODE REVIEW

**Comment by gavofyork** [comment]
they should click through the license whatever; skip button should only be there after that.

**Comment by derhuerst** [comment]
@gavofyork It is the case. The "skip" button is shown in the "create an account" step.

File: js/src/modals/FirstRun/firstRun.js [hunk 1]
```diff
@@ -130,13 +133,26 @@ export default class FirstRun extends Component {
...
+            key='create'
             disabled={ !canCreate }
-            onClick={ this.onCreate } />
-        );
+            onClick={ this.onCreate }
+          />
+        ];
+        if (hasAccounts) {
+          buttons.unshift(
+            <Button
```
Comment (jacogr) [inline js/src/modals/FirstRun/firstRun.js]: Would have preferred the labels/hint/text changes into `FormattedMessage` while we are in the component. (Saves work later, especially when adding new features). 

Not a crisis here though, so will just comment on it - small enough change overall.

**Comment by jacogr** [comment]
Additional prop verification issues introduced: (2x)

vendor.js:6509 Warning: Failed prop type: Invalid prop `showFirstRun` of type `number` supplied to `Container`, expected `boolean`.
    in Container (created by Application)
    in Application (created by Connect(Application))
    in Connect(Application) (created by RouterContext)
    in RouterContext (created by Router)
    in Router (created by MainApplication)
    in MainApplication
    in IntlProvider (created by ContextProvider)
    in ContextProvider

**Comment by derhuerst** [comment]
> Additional prop verification issues introduced:

This should only be because there's still `0`/`1` in your localStorage. After reload, this should be set to `false`/`true` and therefore gone.

**Comment by jacogr** [comment]
Yes, you are right, it is a once-off

**Pull request merged**